scripts/db-setup.sql

```diff
@@ -116,4 +116,96 @@ VALUES
   ('000007', 'We’re proud of the quality of our code! It is clean, easy to read, and has great test coverage.', 'quality'),
   ('000008', 'We get stuff done really quickly.No waiting, no delays.', 'speed'),
   ('000009', 'We always get great support & help when we ask for it!', 'support'),
-  ('000010', 'We are in control of our destiny! We decide what to build and how to build it.', 'self-organization');
\ No newline at end of file
+  ('000010', 'We are in control of our destiny! We decide what to build and how to build it.', 'self-organization');
+
+
+
+# Dump of stored procedure
+# ------------------------------------------------------------
+DELIMITER $$
+DROP PROCEDURE IF EXISTS `getResultsBySurvey`; $$
+CREATE PROCEDURE `getResultsBySurvey`(IN team_uuid VARCHAR(32), IN survey_uuid VARCHAR(32))
+BEGIN
```

Comment:
Maybe I'm missing something, but why can't this just be done in a single SQL query instead of creating a temp table and then querying on it again. Couldn't we just join the question table in the avgTable query and get the uuid there and just use that as the query?

Reply:
You can do in a single SQL statement. Because we are still unsure what column to return, single statement is hard to track if you forgot to add to GROUP BY clause. We can change it easily later